src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRentCost = 40;
+
+  const discountForLongPeriod = 50;
+  const longPeriod = 7;
```

Comment:
Better to show in what units you measure the period.
```suggestion
  const longPeriodDays = 7;
```